src/main/java/frc/robot/Util.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+//
+package frc.robot;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/** Add your docs here. */
+public class Util {
+
+    public static Translation3d getNormalTranslation(Translation3d vec){
+        return(new Translation3d(vec.getX()/vec.getNorm(), vec.getY()/vec.getNorm(), vec.getZ()/vec.getNorm()));
+    }
+
+    public static class PIDConstants{
```

Comment:
Seems a little unusual to call this PIDConstants, normally we use this term for, you know, constants. Maybe PIDParameters would be better?

Reply:
Thank you for the suggestion. We changed the name to PIDParameters.